.gitmodules

```diff
@@ -1,3 +1,19 @@
 [submodule "external/monero-project"]
 	path = external/monero-project
-	url = https://github.com/woodser/monero
+	url = https://github.com/nsec1/monero.git
+	branch = fixIncludePaths
```

Comment:
Can the change in your branch, "fix zmq and sodium include dirs search", be generally applied to monero-project and opened as a PR?

If so, the PR should be opened ASAP, and then I can include your PR as a customization in monero-project until it's merged.

Reply:
monero-project/monero#9504